ragen/env/search/server_search.py

```
@@ -0,0 +1,134 @@
+                "score": 0.8
+            },
+            {
+                "title": f"Search Result 2 for '{query}'",
+                "text": f"Additional information related to '{query}' that might be useful for answering the question.",
+                "contents": f"More search results for '{query}': This is another placeholder result showing the expected format.",
+                "score": 0.7
+            }
+        ]
+
+
+def get_server_search_adapter(config) -> Optional[Any]:
```

Ruff (0.14.10)

Dynamically typed expressions (typing.Any) are disallowed in `get_server_search_adapter`

(ANN401)